__tests__/nano_contracts/deserializer.test.ts

```diff
@@ -67,6 +67,26 @@ test('Bytes', () => {
   const deserialized = deserializer.deserializeFromType(serialized, 'bytes');
 
   expect(value.equals(deserialized)).toBe(true);
+
+  const serializedVertex = serializer.serializeFromType(value, 'VertexId');
```

Comment:
chore: This line and some others in this test indicate that there are unused variables. This may indicate a problem in the tests, with missing assertions.

Reply:
You are completely correct. Fixed in https://github.com/HathorNetwork/hathor-wallet-lib/pull/735/commits/3467f416537979a33b7d5a85e859aac018a3bcd5